tests/fs/readdir.test.ts

```diff
@@ -67,9 +67,9 @@ suite('readdir and readdirSync', () => {
 
 	test('readdir returns Dirent recursively', async () => {
 		const entries = await fs.promises.readdir(testDir, { recursive: true, withFileTypes: true });
```

Comment:
Thanks, my only concern is that the output of `readdir` should be deterministic. That is, calling the function with the same input multiple times should always result in the same output.

We need to ensure the output always has the same ordering, to ensure users can reliably use the ordering of the output.